src/styles/styles.js

```diff
@@ -1266,6 +1266,14 @@ const styles = {
         alignItems: 'center',
         zIndex: 10,
     },
+
+    hiddenElementOutsideOfWindow: {
```

Comment:
This should be updated too I think, as we're no longer hiding it outside of the window.

Reply:
I wanted to keep the "OutsideOfWindow" part as this element is still outside of the window (positioned right outside of the top/left corner).

Reply:
Ah, I see. Thanks for pointnng that out.